jupyterhub/spawner.py

```diff
@@ -299,6 +299,15 @@ def options_from_form(self, form_data):
         their server.
         """)
 
+#   template_vars = Dict(
```

Comment:
@minrk Would you rather have this uncommented and default to False?

Thanks @rcthomas.

Reply:
If you like I can resubmit this as a separate, cleaner PR and close this one.  Sorry about the mess... 😄

Reply:
Leave it be for now @rcthomas. No apology needed; we appreciate your effort :cookie:

Reply:
If spawner works, feel free to delete this bit. If you like, you can rebase and squash out the changes, but I have no objection to deleting them in a new commit. Your choice! Either way, it doesn't need to be a new PR.